app/assets/stylesheets/trips.css.scss

```diff
@@ -51,7 +51,11 @@
   position: relative;
   border: solid 3px #363636;
   border-radius: 5px 5px 5px 5px;
-  margin: 15px 15px 15px 15px
+  margin: 15px 15px 15px 15px;
+  box-shadow: 0px 30px 40px ;
```

Comment:
Declaration should not have a space before the terminating semicolon<br>`0px` should be written without units as `0`